datashader/core.py

```diff
@@ -196,11 +196,121 @@ def line(self, source, x, y, agg=None):
             Reduction to compute. Default is ``any()``.
         """
         from .glyphs import Line
-        from .reductions import any
+        from .reductions import any as any_rdn
         if agg is None:
-            agg = any()
+            agg = any_rdn()
         return bypixel(source, self, Line(x, y), agg)
 
+    def trimesh(self, vertices, simplices, mesh=None, agg=None, interp=True):
+        """Compute a reduction by pixel, mapping data to pixels as a triangle.
+
+        >>> import datashader as ds
+        >>> verts = pd.DataFrame({'x': [0, 5, 10],
+        ...                         'y': [0, 10, 0],
+        ...                         'weight': [1, 5, 3]},
+        ...                        columns=['x', 'y', 'weight'])
+        >>> tris = pd.DataFrame({'v0': [2], 'v1': [0], 'v2': [1]},
+        ...                       columns=['v0', 'v1', 'v2'])
+        >>> cvs = ds.Canvas(x_range=(verts.x.min(), verts.x.max()),
+        ...                 y_range=(verts.y.min(), verts.y.max()))
+        >>> cvs.trimesh(verts, tris)
+
+        Parameters
+        ----------
+        vertices : pandas.DataFrame, dask.DataFrame
+            The input datasource for triangle vertex coordinates. These can be
+            interpreted as the x/y coordinates of the vertices, with optional
+            weights for color interpolation. Columns should be ordered
+            corresponding to 'x', 'y', followed by zero or more (optional)
+            columns containing vertex values. The rows need not be ordered, and
+            the data type must be from the set of real numbers.
```

Comment:
What does it mean for a data type to be from the set of real numbers?  The set of real numbers doesn't include any types, but I assume you meant that the column type must be integer or real?If so, that's fine, as I've reworded it, but if not please correct this.